cinema/logic.py

```diff
@@ -0,0 +1,2 @@
+def _params_to_ints(string: str) -> list[int]:
+    return [int(str_id) for str_id in string.split(",")]

```

Comment:
incorrect module name. That is not logic. If you wanna have separate module for such things - It should be in `utils` or something like that